ozaria/site/components/teacher-dashboard/common/_purple-secondary-button.scss

```
@@ -0,0 +1,31 @@
+.purple-secondary-btn {
+  border: 2px solid $purple;
+  border-radius: 4px;
+  text-shadow: unset;
+  @include font-p-3-small-button-text-black;
+  background-image: unset;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)